### ISSUE TITLE
Flaky test BlazorWebTemplate_IndividualAuth_LocalDb

### ISSUE THREAD

**Issue by amcasey** [dotnet/aspnetcore#52736]
## Failing Test(s)

- Templates.Mvc.Test.BlazorTemplateTest.BlazorWebTemplate_IndividualAuth_LocalDb(useProgramMain: True)

## Error Message

```text
Assert.DoesNotContain() Failure
Found:    .db
In value: <Project Sdk="Microsoft.NET.Sdk.Web">


        <PropertyGroup>
          <TargetFramework>net9.0</TargetFramework>
          <Nullable>enable</Nullable>
          <ImplicitUsings>enable</ImplicitUsings>
          <UserSecretsId>aspnet-AspNet.db5wczvqvgbh-eb168c2b-2b5a-4d39-9c46-6be63ac8091b</UserSecretsId>
        </PropertyGroup>
      
        <ItemGroup>
          <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="9.0.0-ci" />
          <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="9.0.0-ci" />
          <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="9.0.0-alpha.1.23607.2" />
          <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="9.0.0-alpha.1.23607.2" />
        </ItemGroup>
      
      </Project>
```

## Stacktrace

<details>

```text
   at Templates.Mvc.Test.BlazorTemplateTest.BlazorWebTemplate_Core(String[] args) in /_/src/ProjectTemplates/test/Templates.Mvc.Tests/BlazorTemplateTest.cs:line 80
```
</details>

## Build

https://dev.azure.com/dnceng-public/public/_build/results?buildId=495483

**Comment by MackinnonBuck** [comment]
Marking as `test-fixed` because this was addressed by #53476

**Comment by amcasey** [comment]
Kusto shows PassCount < TotalRunCount for the last 20 days, but I can't find this test on our [dashboard](https://dev.azure.com/dnceng-public/public/_test/analytics?definitionId=84&contextType=build).
